loops1.js

```diff
@@ -12,9 +12,15 @@ c. Print out each meal.*/
 //a
 let meal = [];
 for (let i = 0; i < protein.length; i++) {
-  for ( j = 0; j < grain.length; j++) {
-    const ingredient = protein[i] + " " + grain[i];
-    meal.push(ingredient);
+  for (let j = 0; j < grain.length; j++) {
+    for (let k = 0; k < vegetable.length; k++) {
+      for (let l = 0; l < beverage.length; l++) {
```

Comment:
Četiri petlje jedan u drugoj su no-no :D

Znači, ja mislim da na kraju trebaš dobiti listu sa jelima:

`["chicken, rice, peas, juice, apple", "pork, pasta, green beans, milk, banana", ...]`

```suggestion
  for(let i = 0; i < meals.length; i++) {
     const meal = protein[i] + ',' + grain[i] + ',' + vegetable[i] + ',' +  beverage[i] + ',' + dessert[i];
     meals.push(meal);
  }
```